.mk/develop.mk

```diff
@@ -23,15 +25,24 @@ run-cli: ## run the CLI, needs additional arguments
 run-server: ## run the app
 	@go run -ldflags "-X main.version=$(shell git describe --abbrev=0 --tags)" -tags '$(BUILDTAGS)' ./cmd/server serve
 
+.PHONY: run-docker-teardown
+run-docker-teardown:
```

Comment:
Nice, can you add a comment after the target so it shows when doing make help correctly 👍